benchmarks/torchvision/main.py

```diff
@@ -131,6 +131,12 @@ def main():
         action="store_false",
         help="do not allow tf32",
     )
+    parser.add_argument(
+        "--tf32",
+        dest="allow_tf32",
+        action="store_true",
+        help="Allow tf32",
+    )
```

Comment:
I don't think we should have both `--tf32` and `--no-tf32`. I'm not sure what the default value for `allow_tf32` even is when both options exist.

Reply:
That is a fair point. In this case, maybe we can have an option `--precision=tf32/f32/f16/bf16` instead? (possibly also subsuming `--amp`)